services/core-web/src/components/noticeOfWork/applications/NOWDocuments.js

```diff
@@ -39,33 +42,65 @@ const propTypes = {
   categoriesToShow: PropTypes.arrayOf(PropTypes.string),
   disclaimerText: PropTypes.string,
   isAdminView: PropTypes.bool,
-  addDescriptionColumn: PropTypes.bool,
-  showPreambleFileMetadata: PropTypes.bool,
-  editPreambleFileMetadata: PropTypes.bool,
   updateNoticeOfWorkApplication: PropTypes.func.isRequired,
+  editNoticeOfWorkDocument: PropTypes.func.isRequired,
   fetchImportedNoticeOfWorkApplication: PropTypes.func.isRequired,
   deleteNoticeOfWorkApplicationDocument: PropTypes.func.isRequired,
   allowAfterProcess: PropTypes.bool,
+  disableCategoryFilter: PropTypes.bool,
+  isStandardDocuments: PropTypes.bool,
+  isFinalPackageTable: PropTypes.bool,
+  isRefConDocuments: PropTypes.bool,
+  isPackageModal: PropTypes.bool,
+  showDescription: PropTypes.bool,
+  applicationDelay: PropTypes.objectOf(PropTypes.string).isRequired,
 };
 
 const defaultProps = {
   selectedRows: null,
   categoriesToShow: [],
   disclaimerText: "",
   isAdminView: false,
-  addDescriptionColumn: true,
-  showPreambleFileMetadata: false,
-  editPreambleFileMetadata: false,
   allowAfterProcess: false,
+  disableCategoryFilter: false,
+  isFinalPackageTable: false,
+  isStandardDocuments: false,
+  isRefConDocuments: false,
+  isPackageModal: false,
+  showDescription: false,
 };
 
+export const RenderNowDocumentsTableExpandIcon = (rowProps) => (
+  <div>
+    {rowProps.expanded ? (
+      <Tooltip title="Click to hide amendment history." placement="right" mouseEnterDelay={1}>
+        <MinusSquareFilled className="icon-lg--lightgrey" />
```

Comment:
Update tooltip to mention documents and not amendments

Reply:
updated